ha/core/controllers/pcs/node_controller.py

```diff
@@ -114,7 +119,27 @@ def start(self, nodeid: str) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        _res = self.nodes_status([nodeid])
```

Comment:
start should return 

 {"status": "", "msg":""}
status is InProgress as it is async

Reply:
done